packages/bcer-data-portal/app/src/util/bcRouteLink.util.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  BCGeocoderAutocompleteData,
+  BusinessLocation,
+  RouteOptions,
+} from '@/constants/localInterfaces';
+
+export class BcRouteLinkBuilder {
+  readonly baseLink = 'https://router.api.gov.bc.ca';
```

Comment:
I think we might have to prepend this to the url on the backend side as well in addition to the api key. Otherwise you could still retrieve the api key by passing in a url you own to the endpoint

Reply:
Or could even just add in a check to make sure that only requests to router.api.gov.bc.ca are allowed